python_scripts/video_parallel_plot.py

```diff
@@ -0,0 +1,129 @@
+# %% [markdown]
+# # Hyperparameter tuning visualization
+
+# %% [markdown]
+# In the previous notebook we showed how to implement a randomized
+# search for tuning the hyperparameters of a `HistGradientBoostingClassifier`
+# to fit the `adult_census` dataset. In practice, a randomized hyperparameter
+# search is usually run with a large number of iterations.
+
+# %% [markdown]
+# In order to avoid the computational cost and still make a decent analysis,
+# we load the results obtained from a similar search with 200 iterations.
+
+# %%
+import pandas as pd
+
+cv_results = pd.read_csv("../figures/randomized_search_results.csv", index_col=0)
+
+cv_results.head()
+
+# %% [markdown]
+# We define a function to remove the prefixes in the hyperparameters
+# column names.
+
+# %%
+def shorten_param(param_name):
+    if "__" in param_name:
+        return param_name.rsplit("__", 1)[1]
+    return param_name
+
+
+# %%
+cv_results = cv_results.rename(shorten_param, axis=1)
+cv_results.head()
```

Comment:
I find it more informative to have the default display of dataframes instead of `.head()`.

```suggestion
cv_results
```